packages/calcite-components/src/components/tab-title/tab-title.scss

```diff
@@ -1,3 +1,12 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-tab-end-icon-color: Specifies the component's `iconEnd` color. Fallback to `--calcite-icon-color`.
```

Comment:
Would this make more sense as `--calcite-tab-icon-start-color` and `--calcite-tab-icon-end-color` instead of `start/end-icon`? 

That would match the property names of `iconStart` and `iconEnd` (not `startIcon` / `endIcon`)?

Reply:
That's what I had originally, but changed to follow [accordion-item](https://github.com/Esri/calcite-design-system/pull/10181).

Reply:
Per the naming schema https://github.com/Esri/calcite-design-system/wiki/tokens-naming-schema is should be [domain]-[component]-[element]-[type]-[group] which would be `--calcite-tab-icon-color-start/end`

Reply:
Based on ☝️, we'll need to correct `accordion-item`'s props too. cc @geospatialem 

@alisonailea Can you update the schema diagram to include start/end as examples?

Reply:
Added https://github.com/Esri/calcite-design-system/issues/10894 to next month's December milestone for the `accordion-item` token refactor. cc @alisonailea

Reply:
Updated the schema diagram to include start/end as examples

Reply:
@alisonailea @macandcheese Any additional feedback on the name and PR besides start/end?

Reply:
LGTM !